src/gg.regression.unity.bots/Runtime/Scripts/ClientDashboard/TcpMessage.cs

```diff
@@ -17,17 +18,22 @@ public enum TcpMessageType
         
         Ping,
         
-        // client requests to play a sequence with the given resourcePath
+        // client requests to play a resource with the given resourcePath
         PlaySequence,
+        PlaySegment,
+        
+        // stops any currently-running sequence/segments
+        StopReplay,
         
         // =====================
         // server -> client
         // =====================
         
         Pong,
         
-        // info about the available sequences for this game instance
+        // info about the available file-based resources for this game instance
         AvailableSequences,
+        AvailableSegments,
         
         // info about the currently-running sequence (or segment)
         ActiveSequence,
```

Comment:
Future (non-blocking), I have a task to create a status manager to better isolate the current status of which segment we're on, what the current error state is, whether exploration is active, and the current exploration error state.  We'll want to be able to sync that to the client, but I wanted to put this comment here for visibility into what is coming

Reply:
Yeah, would love to adopt Addison's mockups into this dashboard. We have a lot more freedom here to show detailed information about the current segment, progression through a sequence, any errors, etc.